config.json

```diff
@@ -1,7 +1,7 @@
 {
-    "jsonRpcUrl": "http://localhost:8545",
+    "jsonRpcUrl": "http://ganache:8545",
     "host": {
-        "name": "localhost",
+        "name": "0.0.0.0",
```

Comment:
this will expose your private ip address and anyone can then make a call to your blockchain

Reply:
That host refers to the PISA service, which we map to the ip of the container so that we can access it from outside the container. The docker container for ganache does this same thing though, so we'll have to check that we're not exposing ganache publicly - if we are it should be firewalled. This tutorial does a similar thing.

Reply:
cool